recipes/hole2/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "hole2" %}
+{% set version = "2.3.0" %}
+{% set build_number = environ.get('GIT_DESCRIBE_NUMBER', '0') %}
```

```suggestion
{% set build_number = 0 %}
```

The build number should never be connected to the source versioning. It's used to track changes in the recipe for the _same_ source, so this defeats the purpose. Can you please revert to just `0`?

---

fixed in latest commit